src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMoviesQuery = session.createQuery("from Movie", Movie.class);
+            return getAllMoviesQuery.getResultList();
```

Comment:
```suggestion
            return session.createQuery("from Movie", Movie.class).getResultList();
```
just a suggestion, not necessary